### ISSUE TITLE
Include image alt text in post search

### ISSUE THREAD

**Issue by andromxda** [LemmyNet/lemmy#5445]
### Requirements

- [x] Is this a feature request? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [x] Did you check to see if this issue already exists?
- [x] Is this only a feature request? Do not put multiple feature requests in one issue.
- [x] Is this a backend issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo for UI / frontend issues.
- [x] Do you agree to follow the rules in our [Code of Conduct](https://join-lemmy.org/docs/code_of_conduct.html)?

### Is your proposal related to a problem?

Images are hard to find, they often aren't well described in the post title or body. The only field that is really meant for image descriptions is the alt text field.

### Describe the solution you'd like.

Image posts should be returned if the search query matches the alt text of an image

### Describe alternatives you've considered.

Implement a dedicated image search feature?

**Comment by Nutomic** [comment]
This is easy to add, only requires a single line added here: https://github.com/LemmyNet/lemmy/blob/main/crates/db_views/src/post/post_view.rs#L491

**Comment by DraconicNEO** [comment]
> This is easy to add, only requires a single line added here: https://github.com/LemmyNet/lemmy/blob/main/crates/db_views/src/post/post_view.rs#L491

Seems quick and simple enough so I sent a pull request. I did have to do more than one line but still seemed simple enough. What do you think? Did I do good?

**Comment by DraconicNEO** [comment]
@Nutomic @andromxda I think this issue can be closed as completed now, the function is implemented in my pull request #5449 all it needs is someone with write access to merge it.

**Comment by Nutomic** [comment]
It will be closed automatically once the PR is merged.
